client/src/ui/PageFooter.tsx

```diff
@@ -0,0 +1,172 @@
+"use client";
+import Link from "next/link";
+import Image from "next/image";
+import { Typography, Divider } from "@mui/material";
+
+/*
+This Page Footer will be responisble for rendering the footer of all pages in the project
+Except for the Login Page and Sign Up Page
+*/
+
+export const PageFooter = () => {
+  return (
+    <footer
+      style={{
+        backgroundColor: "#f9f9fc",
+        padding: "20px 40px",
+        display: "flex",
+        justifyContent: "space-evenly",
+        flexWrap: "wrap",
+      }}
+    >
+      {/* Left Section */}
+      <div
+        style={{
+          display: "flex",
+          flexWrap: "nowrap",
+        }}
+      >
+        <Link
+          href="#"
+          style={{ margin: "0 40px", color: "#000", textDecoration: "none" }}
```

Comment:
No errors, but make sure to run the Prettier formatting

Reply:
That is Prettier formatting.

Reply:
oh lol wth, I don't know why sometimes it does all in a single line or in multile lines